packages/crayons-i18n/package.json

```diff
@@ -38,6 +38,7 @@
     "@rollup/plugin-dynamic-import-vars": "^1.4.1",
     "@rollup/plugin-node-resolve": "^11.1.0",
     "@rollup/plugin-typescript": "^8.1.0",
+    "@types/estree": "^0.0.51",
```

Comment:
do we need this dependency here ?

Reply:
This was necessary when we had the plugin as a package. Removed this now.